### PR TITLE
Sharding release 2019Q2: sharding enable, blockchain speedup

### PR DESCRIPTION
**Pull request by 0486**

### CODE REVIEW

File: apl-conf/src/main/resources/conf-tn1/chains.json [hunk 1]
```diff
@@ -65,12 +65,7 @@
         "blockTime": 10,
         "maxBlockTimeLimit": 60,
         "minBlockTimeLimit": 7,
-        "maxBalance": 30000000000,
-        "consensusSettings": {
-          "adaptiveForgingSettings": {
-            "enabled": "false"
```
Comment (AndrewBoyarsky) [inline apl-conf/src/main/resources/conf-tn1/chains.json]: By default adaptive forging is disabled

File: apl-conf/src/main/resources/conf-tn1/chains.json [hunk 2]
```diff
@@ -134,12 +129,7 @@
         "blockTime": 2,
         "maxBlockTimeLimit": 5,
         "minBlockTimeLimit": 1,
-        "maxBalance": 10000000000,
-        "consensusSettings": {
-          "adaptiveForgingSettings": {
-            "enabled": "false"
```
Comment (AndrewBoyarsky) [inline apl-conf/src/main/resources/conf-tn1/chains.json]: same as above

File: apl-conf/src/main/resources/conf-tn1/chains.json (hunk 1 above)
Comment (0486) [reply apl-conf/src/main/resources/conf-tn1/chains.json]: ?

**Comment by vgiglio** [comment]
Can't help but leave a comment. Thank You for your hard work and dedication from all of us Apollo Investors.

**Comment by mitymouse1371** [comment]
seriously thanks a lot. we've all made history. some of us do look forward
to your tech
“*In war*, *there are no unwounded soldiers*.”
                 Quote by Jose Narosky

**Pull request merged**